docs/src/main/sphinx/language/reserved.rst

```diff
@@ -69,6 +69,7 @@ Keyword                        SQL:2016      SQL-92
 ``RIGHT``                      reserved      reserved
 ``ROLLUP``                     reserved
 ``SELECT``                     reserved      reserved
+``SKIP``                       reserved      reserved
```

Comment:
move to the other related doc PR .. and also note that we will need to make sure to have a release notes entry about this additional keyword

Reply:
It cannot be removed. This PR does not pass tests without it.

Reply:
Ok .. fair enough. Just make sure we add this in the release notes